src/main/java/ithillel/lesson13/ArrayTask.java

```diff
@@ -0,0 +1,41 @@
+package ithillel.lesson13;
+
+import java.util.concurrent.RecursiveTask;
+
+public class ArrayTask extends RecursiveTask<double[]> {
+    private final double[] data;
+    private final int start;
+    private final int end;
+
+    public ArrayTask(double[] data, int start, int end) {
+        this.data = data;
+        this.start = start;
+        this.end = end;
+    }
+
+    @Override
+    protected double[] compute() {
+        if(end - start < 50){
+            System.out.println("Starting... " + Thread.currentThread().getName());
+            for(int i = start; i <= end; i++){
```

Comment:
```suggestion
            for(int i = start; i < end; i++){
```
тоді простіше з індексами працювати (не потрібно буде робити плюс або мінус 1).